packages/date-picker/docs/DatePicker.stories.tsx

```diff
@@ -23,88 +30,154 @@ export const KaizenDefault = props => {
-      onChange={onDayChange}
-      isDisabled
-    />
+    <>
+      <StoryWrapper isReversed={isReversed}>
+        <StoryWrapper.RowHeader
+          headings={["Default", "Selected Value", "Disabled"]}
+        />
+        <StoryWrapper.Row rowTitle="Input">
+          <DatePicker
+            id="datepicker-input-default"
+            labelText="Label"
+            value={selectedDate}
+            onChange={onDayChange}
+          />
+          <DatePicker
+            id="datepicker-input-selected"
+            labelText="Label"
+            value={new Date(2022, 1, 5)}
+            onChange={onDayChange}
+          />
+          <DatePicker
+            isDisabled
+            id="datepicker-input-disabled"
+            labelText="Label"
+            value={selectedDate}
+            onChange={onDayChange}
+          />
+        </StoryWrapper.Row>
+      </StoryWrapper>
+      <StoryWrapper isReversed={isReversed}>
+        <StoryWrapper.RowHeader
+          headings={["Selected Date", "Focused Date", "Disabled Dates"]}
+        />
+        <StoryWrapper.Row rowTitle="Calendar">
+          <CalendarTemplate value={new Date(2022, 1, 5)} />
+          <CalendarTemplate
+            value={new Date(2022, 0, 5)}
+            initialMonth={new Date(2022, 0, 5)}
+          />
+          <CalendarTemplate
+            disabledDays={[
+              new Date(2022, 1, 15),
+              { after: new Date(2022, 1, 17) },
+            ]}
+          />
+        </StoryWrapper.Row>
+      </StoryWrapper>
+    </>
   )
 }
-DisabledInput.parameters = { chromatic: { disable: false } }
 
-export const DisabledInputWithValue = () => {
+export const StickerSheetDefault = StickerSheetTemplate.bind({})
+StickerSheetDefault.storyName = "Sticker Sheet (Default)"
+StickerSheetDefault.parameters = { chromatic: { disable: false } }
+StickerSheetDefault.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement)
+  const focusedDate = canvas.getByLabelText("Wed Jan 19 2022")
+  await userEvent.click(focusedDate, undefined, {
+    skipPointerEventsCheck: true,
+  })
+}
+
+export const FormExample = props => {
   const [selectedDate, setSelectedDate] = useState<Date | undefined>()
 
   const onDayChange = (day: Date) => {
     setSelectedDate(day)
   }
+  const options = [
+    { value: "Lloyd", label: "Lloyd" },
+    { value: "Nat", label: "Nat", isDisabled: true },
+    { value: "Rafa", label: "Rafa" },
+    { value: "Elaine", label: "Elaine" },
+    { value: "Julio", label: "Julio" },
+    { value: "Priyanka", label: "Priyanka" },
+    { value: "Prince", label: "Prince" },
+    { value: "Charith", label: "Charith" },
+    { value: "Nick", label: "Nick" },
+  ]
 
   return (
-    <DatePicker
-      id="make-me-unique-5"
-      labelText="Label"
-      value={new Date(2022, 1, 14)}
-      onChange={onDayChange}
-      isDisabled
-    />
+    <>
+      <p>
+        This story is used to test the z-index in comparision to other pop up
```

Comment:
Is the intention to keep this story? If it's not going on Chromatic and it's just testing to see if z-index works, I'd say it's just adding clutter?

Reply:
yeah I would agree, I will remove :)

Reply:
If it shows the z-index works, I would send it to Chromatic instead of removing it :)

Reply:
Its a weird one -  Since both of the components don't have a prop to set the initial state to be opened, to get into the state where the popups overlap you need to click a few times back and forth. We would need to use userEvents and testing-library to get it to open which will take a bit of work.

Reply:
Im thinking for now it is not worth the effort in terms of what is required for this card and since this PR is required for the next date picker card. The z-index was a _nice to have_ extra of this card so I can note it in another card.

Reply:
I actually had requested the story as something to have when Lloyd was implementing the z-index fix, so that we could have a visual test in Chromatic.

The issue at the time was that the DatePicker, when open, was actually sitting below some other form elements (I think it was a text input?). Even if it doesn't work for Chromatic yet, it's worth us being able to verify that it's not opening up under other elements.

Reply:
I'll create another card for it and i'll add your comments above - Ideally this PR gets merged as we want to use chromatic for the next card that potentially breaks the focus styling.

Reply:
Sure, go for it :)